Content.Server/Chat/Systems/ChatSystem.cs

```diff
@@ -737,6 +741,17 @@ private string SanitizeMessagePeriod(string message)
         return message;
     }
 
+    public string SanitizeMessageReplaceWords(string message)
+    {
+        if (string.IsNullOrEmpty(message)) return message;
+
+        var msg = message;
+
+        msg = _wordreplacement.ApplyReplacements(msg, "chatsanitize");
```

Comment:
make "chatsanitize" a const string with the `[ValidatePrototypeId<AccentPrototype>]` attribute on it.

Reply:
hopefully did it, is it good?